management/persistence.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Persistence
+nav_order: 1
+description: How FalkorDB writes data to disk
+parent: Management
+---
+
+Persistence refers to the writing of data to durable storage, such as a solid-state disk (SSD). FalkorDB provides a range of persistence options. These include:
+
+* **RDB** (FalkorDB Database): RDB persistence performs point-in-time snapshots of your dataset at specified intervals.
+* **AOF** (Append Only File): AOF persistence logs every write operation received by the server. These operations can then be replayed again at server startup, reconstructing the original dataset. Commands are logged using the same format as the FalkorDB protocol itself.
```

Comment:
"Commands are logged using the same format as the FalkorDB protocol itself."
Not sure what's the value of stating this, moreover this is not necessarily true, in case a query is replicated / persisted using EFFECTS.